src/update/balrog.cpp

```diff
@@ -483,7 +500,8 @@ bool Balrog::install(const QString& filePath) {
   connect(process,
           QOverload<int, QProcess::ExitStatus>::of(&QProcess::finished), this,
           [this, process](int exitCode, QProcess::ExitStatus) {
-            logger.debug() << "Installation completed - exitCode:" << exitCode;
+            logger.debug() << "Async installation activated - exitCode:"
```

Comment:
Debug errors don't appear in logs for non-devs, right? If yes, then would this be useful to have in logs?